src/App.tsx

```diff
@@ -5,49 +5,70 @@ import {
   Card,
   CardContent,
   CardMedia,
-  ExpansionPanel, 
-  ExpansionPanelSummary, 
-  ExpansionPanelDetails, 
   Grid,
+  InputBase,
   Toolbar,
   Typography,
 } from '@material-ui/core';
 import CameraIcon  from '@material-ui/icons/PhotoCamera';
+import SearchIcon  from '@material-ui/icons/Search';
 import './App.css';
-import { withStyles } from '@material-ui/core/styles';
+import { withStyles, createStyles, Theme } from '@material-ui/core/styles';
+
 const heliumApi = 'https://heliumint.azurewebsites.net/api/';
 const cors = 'https://cors-anywhere.herokuapp.com/';
 
+type Movie = {
+  movieId: string,
+  type: string, 
+  title: string, 
+  year: string, 
+  runtime: number,
+  genres: string[], 
+}
+
+type Genre = {
+  id: string,
+  name: string,
+}
+
+type Actor = {
+  id: string,
+  name: string,
+}
+
+
 const divStyle = {
   color: 'blue',
   backgroundColor: 'blue',
 };
 
+interface IProps {
+  superhero: string;
```

Comment:
This all looks good, but what does `superhero` mean here?

Reply:
lol, an artifact of overzealous copy/pasting